api/src/main/resources/workbench-api.yaml

```diff
@@ -159,14 +159,29 @@ paths:
     get:
       tags:
       - cdrVersions
-      description: Returns all curated data repository (CDR) versions that the user
-        has access to
+      description: >
```

Comment:
No UI changes are made here: it continues to reference the existing/deprecated `cdrVersions` endpoint.  We will keep this endpoint around for at least one release cycle after the UI is pointed to the new `cdrVersionsByTier` endpoint, to avoid release version mismatch.